remote/src/lib.rs

```diff
@@ -171,21 +171,33 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
                         );
                     }
                 }
-            } else {
-                // Delete by peer name.
-                for (name, peer) in git::remotes(&repo)? {
-                    if let Some(person) = project::person(&storage, urn.clone(), &peer)? {
-                        if person.subject().name.to_string() == remote {
-                            repo.remote_delete(&name)?;
-                            term::success!(
-                                "Remote {} {} removed",
-                                term::format::highlight(&remote),
-                                term::format::dim(format!("{:?}", name)),
-                            );
-                        }
+
+                return Ok(())
+            }
+
+            // Delete by peer name.
+            let mut x = None;
+            for (name, peer) in git::remotes(&repo)? {
+                if let Some(person) = project::person(&storage, urn.clone(), &peer)? {
+                    if person.subject().name.to_string() == remote {
+                        x = Some((name, &remote));
+                        break;
```

Comment:
Why was this block of code moved out of the if/else above?

Reply:
It was a vain attempt at reducing the code's indentation level.  In the end, I preserve the if/else structure in the proposed change.